vllm/model_executor/models/commandr.py

```diff
@@ -271,8 +275,8 @@ def __init__(self, *, vllm_config: VllmConfig, prefix: str = ""):
                                                    config.hidden_size)
         self.start_layer, self.end_layer, self.layers = make_layers(
             config.num_hidden_layers,
-            lambda prefix: CohereDecoderLayer(config, cache_config,
-                                              quant_config),
+            lambda prefix: CohereDecoderLayer(
+                config, cache_config, quant_config, prefix=prefix),
```

Comment:
Since so many layers just pass through `cache_config`, `quant_config` and `prefix`, I think we can have a more convenient version of `make_layers` that directly forwards those arguments without having to define our own lambda function.

Reply:
that can be another pr, if you want, to refactor the way we create layers.